src/components/interactive-lab-callout/interactive-lab-callout.module.css

```diff
@@ -11,11 +11,17 @@
 		display: flex;
 		gap: 32px;
 	}
+
+	@nest html[data-theme='dark'] & {
+		background-color: var(
+			--token-color-foreground-strong
+		); /** Needed to supercede the dark mode card .root dark mode background  */
+	}
```

Comment:
This is the second time i've seen an issue with scope using the overrides method. The current CSS with this component should work, as it is the same value that we've put here -- it should be the same for dark and light mode. But this is wrapped in a card and therefore the card background color override was rendering instead of the bg color that was set on line 8 in this file. So we have to redefine the background color for dark mode here. 

Another reason to _really_ avoid overrides. When we `@nest html` this supercedes even child styles that are scoped to those components.

Reply:
Trying to make sure I'm following -- are you saying we should be avoiding `@nest html`?

Reply:
I think we should just be aware of potential scope issues when we do use it, given the behavior here. And we for that reason we should limit overrides. 

To clarify what happened here: I had to redefine the background color for this component even though it was already set with that set value since the `Card` (which is a parent here) dark mode background-color override was taking precedent over this component's styles.

Reply:
Can you elaborate on what you mean by `override`? I'm fully understanding the technical CSS issue -- we only need this `@nest` here because there's one in `Card`.

Reply:
Yes exactly, I understand why the specificity is taking precedent, I'm just noting it as a side-effect of doing theming overrides that we need to keep in mind since we're so used to scoped css modules.

Reply:
Ok sweet we're on the same page then! I thought when you said "The current CSS with this component should work" in your first comment, maybe you were worried about this being a bug. 💟